Rakefile

```diff
@@ -7,3 +7,8 @@ Rails.application.load_tasks
 def is_running_migration?
   Rake.application.top_level_tasks.include?("db:migrate")
 end
+
+Rake::Task["db:schema:load"].clear
+task "db:schema:load" do
+  Rake::Task["db:structure:load"].invoke
```

Comment:
Why do we need to redefine this task if we manually call the structure load in `heroku-setup` ?